pkg/sippyserver/server.go

```diff
@@ -793,6 +794,12 @@ func (s *Server) jsonJobRunRiskAnalysis(w http.ResponseWriter, req *http.Request
 		}
 
 		// TODO: get the count of tests that were run overall
+		// select id, count(*) from prow_job_run_tests where prow_job_run_id = jobRunID
+		jobRunTestCount, err = query.JobRunTestCount(s.db, jobRunID)
+		if err != nil {
+			log.Errorf("Error getting job run test count %v", err)
```

Comment:
```suggestion
			log.WithError(err).Error("Error getting job run test count")
```

This one's a NIT if you've got the error in the message anyhow but structured logging is still good for us to keep in mind.